src/modules/dashboard/components/HookedTokenIcon/HookedTokenIcon.tsx

```diff
@@ -24,76 +26,67 @@ interface Props {
 
   /** Optional name for the icon title */
   name?: string;
+
+  /** If provided than icon is display instead of Avatar */
+  iconName?: string;
 }
 
-const checkSVG = (fileName: string) =>
-  fileName &&
-  fileName.substring(fileName.length - 3, fileName.length) === 'svg';
+const TOKEN_LOGOS_REPO_URL =
+  'https://raw.githubusercontent.com/trustwallet' +
+  '/assets/master/blockchains/ethereum/';
 
-const loadTokenImages = async (logo): Promise<ImageType> =>
-  import(
-    /* webpackMode: "eager" */ `../../../../../node_modules/eth-contract-metadata/images/${logo}`
-  );
+const loadTokenImages = async (address: Address): Promise<Response> => {
+  let tokenImageUrl = `${TOKEN_LOGOS_REPO_URL}${address}/logo.png`;
+  if (address === AddressZero) {
+    tokenImageUrl = `${TOKEN_LOGOS_REPO_URL}info/logo.png`;
+  }
+  return fetch(tokenImageUrl);
+};
```

Comment:
I think there should be an extra prop to be passed down to this component: `dontFetch` that should prevent fetching of the token image. _(it should default to `false`)_

This way we can conditionally prevent fetches that we know are going to fail _(eg: for native tokens that won't have an icon)_

![Screenshot from 2020-12-10 21-50-01](https://user-images.githubusercontent.com/1193222/101822343-d6e99280-3b31-11eb-921f-0c83d29ced11.png)